nixos/modules/services/video/wivrn.nix

```diff
@@ -0,0 +1,122 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```
bad practice, please avoid
https://nix.dev/guides/best-practices#with-scopes
You're welcome to use `inherit` instead